expression/builtin/time.go

```diff
@@ -309,6 +309,13 @@ func builtinCurrentDate(args []interface{}, ctx map[interface{}]interface{}) (in
 		Type: mysql.TypeDate, Fsp: 0}, nil
 }
 
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_curtime
+func builtinCurrentTime(args []interface{}, ctx map[interface{}]interface{}) (interface{}, error) {
+	return mysql.Time{
+		Time: time.Now(),
+		Type: mysql.TypeDuration, Fsp: 0}, nil
```

Comment:
@siddontang `mysql.TypeTimestamp` is for time format: `YYYY-MM-DD HH:MM:SS` and it already has corresponding time functions (`CURRENT_TIMESTAMP()` and `NOW()`).

Actually what I really want to use here is `mysql.TypeTime`, but when I check the code in `mysql/type.go`, the `TypeTime` is renamed to `TypeDuration` and the comment says it has to resolve the conflict with Go type time.

I know it is a bit of confusing but finally I have to adhere to the existing code.

If you have some other workarounds, please let me know.